### PR TITLE
SOC-2096 | pop-over content works with every kind of click

### PR DESCRIPTION
**Pull request by rafalkalinski**
We just wanted to have a possibility to open links placed in pop-over in a separate window.

@kvas-damian CR please

### CODE REVIEW

**Comment by kvas-damian** [comment]
LGTM

File: addon/components/pop-over.js [hunk 1]
```diff
@@ -177,8 +176,7 @@ export default Ember.Component.extend({
     set(this, 'pressed', false);
     var targets = get(this, 'targets');
     var element = get(this, 'element');
-    var clicked = isSimpleClick(evt) &&
-      (evt.target === element || $.contains(element, evt.target));
+    var clicked = (evt.target === element || $.contains(element, evt.target));
```
Comment (bkoval) [inline addon/components/pop-over.js:179]: Does it mean popover will open even on a single right click?

Comment (rafalkalinski) [inline addon/components/pop-over.js:179]: Nein.
It opens just like before.

Comment (bkoval) [inline addon/components/pop-over.js:179]: 💛

**Comment by bkoval** [comment]
LGTM

**Pull request merged**